### ISSUE TITLE
Add accessibility user settings

### ISSUE THREAD

**Issue by buddh4** [humhub/humhub#4943]
Some ui elements as for example the focus richtext menu which is only visible once the richtext is focused do provide a poor keyboard accessibility. In this example, the richtext menu should be visible and focusable prior of focusing the richtext input. It would be great if there was a user setting for enabling/disabling specific accessibility modes. In a first example only for `keyboard access` related issues, later maybe `low vision` etc.

Another use-case for this would be enabling/disabling `tab` in richtext code blocks by `keyboard access` user setting. I'am sure on analysis there are many ui components, which could be improved in terms of accessibility with this setting.

Related issues:

- https://github.com/humhub/humhub-prosemirror/issues/43
- https://github.com/humhub/humhub-prosemirror/issues/30
